### ISSUE TITLE
Cannot access presentations with NVDA on power point viewer

### ISSUE THREAD

**Issue by Fatihyuksek** [nvaccess/nvda#9349]
### Steps to reproduce:
open power point viewer. You will notice that you cannot read the presentations with NVDA
### Actual behavior:
NVDA does not read the contents of any slide with power point viewer
### Expected behavior:
NVDA reads any slide automatically upon pressing space or some other key
#### NVDA installed/portable/running from source:
no
#### NVDA version:
2018.4
#### Windows version:
windows 7
#### Name and version of other software in use when reproducing the issue:
power point viewer
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by leonardder** [comment]
I assume power point viewer works similar to Power Point's view mode, invoked when pressing F5 in Power Point?

**Comment by DrSooom** [comment]
> PowerPoint Viewer was retired on April 30, 2018. It's no longer available for download.

Source: [Microsoft Office Support – View a presentation without PowerPoint](https://support.office.com/en-us/article/view-a-presentation-without-powerpoint-2f1077ab-9a4e-41ba-9f75-d55bd9b231a6)

**Comment by Fatihyuksek** [comment]
I downloaded this from another website. I have office starter and it doesn’t include power point in it. I need this for my education in univercity

iPhone’umdan gönderildi

Daniel Mayr <notifications@github.com<mailto:notifications@github.com>> şunları yazdı (7 Mar 2019 15:58):


PowerPoint Viewer was retired on April 30, 2018. It's no longer available for download.

Source: Microsoft Office Support – View a presentation without PowerPoint<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fsupport.office.com%2Fen-us%2Farticle%2Fview-a-presentation-without-powerpoint-2f1077ab-9a4e-41ba-9f75-d55bd9b231a6&data=02%7C01%7C%7C3b4d119b9553455bfa1708d6a2fc8f5a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636875602946105507&sdata=AF5T5tKRxL3vpuJVxi208DqmVuzzSe%2FN7PMwZMwJJ5U%3D&reserved=0>

**Comment by DrSooom** [comment]
@Fatihyuksek: Which version of the PowerPoint Viewer are you using?

Sadly LibreOffice Impress 6.2.1 seems not to work really well together with NVDA in the presentation mode too. (I tested this right now.)

**Comment by Adriani90** [comment]
Given the powerpoint viewer is discontinued since many years, I don't expect any accessibility updates for it anymore. Closing as abandoned.
